sqlserver/README.md

```diff
@@ -1,32 +1,59 @@
-# Sqlserver Integration
+# Microsoft SQL Server Check
 
-## Overview
+# Overview
 
-Get metrics from sqlserver service in real time to:
+This check lets you track the performance of your SQL Server instances. It collects metrics for number of user connections, rate of SQL compilations, and more.
 
-* Visualize and monitor sqlserver states
-* Be notified about sqlserver failovers and events.
+You can also create your own metrics by having the check run custom queries.
 
-## Installation
+# Installation
 
-Install the `dd-check-sqlserver` package manually or with your favorite configuration manager
+The SQL Server check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your SQL Server instances. If you need the newest version of the check, install the `dd-check-sqlserver` package.
 
-## Configuration
+# Configuration
 
-Edit the `sqlserver.yaml` file to point to your server and port, set the masters to monitor
+Create a file `sqlserver.yaml` in the Agent's `conf.d` directory:
 
-## Validation
+```
+init_config:
 
-When you run `datadog-agent info` you should see something like the following:
+instances:
+  - host: <SQL_HOST>,<SQL_PORT>
+    username: <SQL_ADMIN_USER>
+    password: <SQL_ADMIN_PASSWORD>
+    connector: odbc # alternative is 'adodbapi'
+    driver: SQL Server
+```
 
-    Checks
-    ======
+See the [example check configuration](https://github.com/DataDog/integrations-core/blob/master/sqlserver/conf.yaml.example) for a comprehensive description of all options, including how to use custom queries to create your own metrics.
 
-        sqlserver
-        -----------
-          - instance #0 [OK]
-          - Collected 39 metrics, 0 events & 7 service checks
+Restart the Agent to start sending SQL Server metrics to Datadog.
 
-## Compatibility
+# Validation
 
-The sqlserver check is compatible with all major platforms
+Run the Agent's `info` subcommand and look for `sqlserver` under the Checks section:
+
+```
+  Checks
+  ======
+    [...]
+
+    sqlserver
+    -------
+      - instance #0 [OK]
+      - Collected 26 metrics, 0 events & 1 service check
+
+    [...]
+```
+
+# Compatibility
+
+The sqlserver check is compatible with all major platforms.
```

Comment:
actually, only `windows` and `linux` (see https://github.com/DataDog/integrations-core/blob/master/sqlserver/manifest.json)